src/scripts/audit.js

```diff
@@ -146,8 +146,10 @@ function recordAdapterCall(context, next, done) {
 		if (bot) {
 			bot.logger.debug(`${TAG}:  recordAdapterCall with context.response.message.isLogged of ` + JSON.stringify(context.response.message.isLogged));
 		}
+
 		if (context.response.message.isLogged === undefined) {
-			if (context.response.message.text) {
+			// ignore messages that don't have text or that come from the user hubot which runs tests
+			if (context.response.message.text && context.response.message.user.name !== 'hubot') {
```

Comment:
could we put a guard in to check that `context.response.message.user` exists before checking the name?